third_party/terraform/website/google.erb

```diff
@@ -146,6 +146,9 @@
       <li<%%= sidebar_current("docs-google-datasource-storage-project-service-account") %>>
         <a href="/docs/providers/google/d/google_storage_project_service_account.html">google_storage_project_service_account</a>
       </li>
+      <li<%= sidebar_current("docs-google-datasource-storage-transfer-project-service-account") %>>
```

Comment:
```suggestion
      <li<%%= sidebar_current("docs-google-datasource-storage-transfer-project-service-account") %>>
```

Super silly change & I miss it every time, but because we're generating a `.erb` file inside a `.erb` file we need to double escape here.

You'll need to make a similar change in https://github.com/GoogleCloudPlatform/magic-modules/pull/1069/files.